apm-collector/apm-collector-baseline/collector-baseline-computing-define/src/main/java/org/skywalking/apm/collector/baseline/computing/Configuration.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2017, OpenSkywalking Organization All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Project repository: https://github.com/OpenSkywalking/skywalking
+ */
+
+package org.skywalking.apm.collector.baseline.computing;
+
+/**
+ * @author zhang-chen
+ */
+public class Configuration {
+
+    /**
+     * how many highest and lowest value will be discarded, when computing average. example: we have eight days original
+     * performance data, if we set discard as four, two highest value and two lowest value will be discarded in order to
+     * ensure the baseline won't disturb by some extreme data. "discard" must be even number
+     */
+    int discard;
```

Comment:
Based on your description, I prefer the `discard` is the exact value of how many highest/lowest values be discarded. Don't need to divide 2. The configuration is based on `application-default.yml`(provided by OpenSkywalking team) and `application.yml`(provided by users). So the even number requirement is meaningless. After all, easier is better, right?